package.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
- [ ] You are not using the readme template given by microverse :fearful: It would be nice if you add that template :+1: to make your project professional :blush:

**Hint: You can get the template from this [link](https://github.com/microverseinc/readme-template/blob/master/README.md) :+1:**